examples/resources/grafana_organization_preferences/resource.tf

```diff
@@ -1,5 +1,5 @@
 resource "grafana_organization_preferences" "test" {
   theme      = "light"
   timezone   = "utc"
-  week_start = "Tuesday"
+  week_start = "tuesday"
```

Comment:
tuesday is an invalid value. We should change that. It has always been wrong

Reply:
Of course. the only values accepted are:  `sunday`, `monday`, `saturday`